shared/teams/new-team/wizard/new-team-info.tsx

```diff
@@ -166,11 +172,17 @@ const NewTeamInfo = () => {
 
         <Kb.Checkbox
           labelComponent={
-            <Kb.Box2 direction="vertical" alignItems="flex-start" style={styles.tallEnoughBox}>
+            <Kb.Box2 direction="vertical" alignItems="flex-start" style={Styles.globalStyles.flexOne}>
               <Kb.Text type="Body">Make it an open team</Kb.Text>
               <Kb.Text type="BodySmall">Anyone can join without admin approval.</Kb.Text>
-              {openTeam && (
-                <Kb.Box2 direction="horizontal" gap="xtiny" alignSelf="flex-start" alignItems="center">
+              {(!Styles.isMobile || openTeam) && (
+                <Kb.Box2
+                  direction="horizontal"
+                  gap="xtiny"
+                  alignSelf="flex-start"
+                  alignItems="center"
+                  className={Styles.classNames('hideableDropdown', {hidden: !openTeam})}
+                >
                   <Kb.Text type="BodySmall">People will join as</Kb.Text>
                   <FloatingRolePicker
```

Comment:
Owners is currently allowed in the dropdown but I think reader,writer,admin are the only valid options for this